.github/workflows/docker-nightly-build.yml

```diff
@@ -0,0 +1,42 @@
+name: Build and push docker nightly
+
+on:
+  workflow_dispatch:
+      inputs:
+      mode:
+        description: 'release/nightly, default is nightly'
+        required: true
+        default: 'nightly'
+
+jobs:
+  nightly-build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'corretto'
+          java-version: 11
+      - name: Login to Docker
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Build and push nightly docker image
+        if: ${{ github.event.inputs.mode == '' || github.event.inputs.mode == 'nightly' }}
+        working-directory: serving/docker
+        run: |
+          DJL_VERSION=$(cat ../../gradle.properties | awk -F '=' '/djl_version/ {print $2}')
+          export TAG=minimal-nightly
+          docker-compose build --build-arg djl_version=${DJL_VERSION}~SNAPSHOT
+          docker-compose push
+      - name: Build and push release docker image
+        if: ${{ github.event.inputs.mode == 'release' }}
+        working-directory: serving/docker
+        run: |
+          DJL_VERSION=$(cat ../../gradle.properties | awk -F '=' '/djl_version/ {print $2}')
+          export TAG=minimal-release
+          docker-compose build --build-arg djl_version=${DJL_VERSION}
+          docker-compose push
```

Comment:
We should create a latest tag which is the same latest version:
see: https://hub.docker.com/r/deepjavalibrary/djl-serving/tags